### ISSUE TITLE
Triggering by ocular surgery

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10930]
**Preferred term label:**
Triggering by ocular surgery

**Synonyms**
na

**Definition (free text, please give PubMed ID)**
Useful as a parent term to classify the presence of a corneal graft (HP:6000096), intraocular lens (HP:500081), a filtering bleb, wound dehiscence, glaucoma implant devices, secondary cataract and so on.

**Parent term (use hpo.jax.org/app)**
HP:12372

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
NA

**Your nano-attribution (ORCID)**
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
We have a relatively new section of terms related to past medical history including  a [Triggered by HP:0025204](https://hpo.jax.org/browse/term/HP:0025204) hierarchy. These terms would be combined with terms that describe the observed phenotypes, e.g., in a [GA4GH phenopacket](https://phenopacket-schema.readthedocs.io/en/latest/). Would this be what you are looking for?

**Comment by daboe01** [comment]
> [@daboe01](https://github.com/daboe01) We have a relatively new section of terms related to past medical history including a [Triggered by HP:0025204](https://hpo.jax.org/browse/term/HP:0025204) hierarchy. These terms would be combined with terms that describe the observed phenotypes, e.g., in a [GA4GH phenopacket](https://phenopacket-schema.readthedocs.io/en/latest/). Would this be what you are looking for?

exactly! than you.

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001448
- label: History of ocular surgery
- parents: [Past surgical history (HP:0430103)](https://hpo.jax.org/app/browse/term/HP:0430103)
- pmid: 
- def: Past medical history of surgical procedure on the eye or its surroundings.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10930
- synonyms:
